src/main/java/edu/kis/vh/nursery/DefaultCountingOutRhymer.java

```diff
@@ -0,0 +1,38 @@
+package edu.kis.vh.nursery;
+
+public class DefaultCountingOutRhymer {
+
+    public static final int DEFAULT_VALUE = -1;
+    public static final int FULL = 11;
```

Comment:
This constant is related to FULL (yet, the name is misinforming). Changing only one leads to an error (inconsistency).